### PR TITLE
Update Uzbek UI localization strings

### PR DESCRIPTION
**Pull request by ozbek**

### CODE REVIEW

File: app/src/main/res/values-uz/strings.xml [hunk 1]
```diff
@@ -90,6 +92,8 @@
     </string>
   <string name="search_hint">Izlash</string>
   <string name="quranSearchType">Suralar</string>
+  <string name="search_full_results">Barcha natijalar</string>
```
Comment (ozbek) [inline app/src/main/res/values-uz/strings.xml:95]: I could not find use case for this string (and one below). From the source logic, it looks like it is only used in the Arabic locale. Maybe put the Arabic translations in the default locale strings and make them non-translatable? What do you think @ahmedre?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -224,8 +224,8 @@
   <string name="prefs_export_summary">Export a copy of bookmarks and tags</string>
   <string name="prefs_page_type_title">Page Type (experimental)</string>
   <string name="prefs_page_type_summary">Select the type of reading pages</string>
-  <string name="prefs_sura_translated_name_summary">Show sura translated name</string>
-  <string name="prefs_sura_translated_name_title">Sura translated name</string>
+  <string name="prefs_sura_translated_name_summary">Show the translation of surah name</string>
```
Comment (ozbek) [inline app/src/main/res/values/strings.xml:227]: Synced with the `surah` usage elsewhere (also minor wording change).

**Comment by ozbek** [comment]
Travis build has failed at `com.quran.labs.androidquran.ui.QuranActivityTest > testClickingOnSuraOnListViewNavigatesToSura[test(AVD) - 5.1.1]` but the test is OK on my local machine :\

File: app/src/main/res/values-uz/strings.xml (hunk 1 above)
Comment (ahmedre) [reply app/src/main/res/values-uz/strings.xml:95]: yeah that sounds reasonable today - but am wondering if we should have something similar for non-Arabic cases where the number of results is a lot more than what we'd show in the suggestions instead.

**Comment by ahmedre** [comment]
jazakumAllah khairan!

**Pull request merged**